pkg/proxy/nftables/proxier.go

```diff
@@ -747,13 +793,104 @@ func (proxier *Proxier) Sync() {
 	proxier.syncRunner.Run()
 }
 
+func (proxier *Proxier) syncAcceleratedInterfaces() {
+	minInterval := 5 * time.Second
+	maxInterval := 1 * time.Minute
+	rateLimiter := rate.NewLimiter(rate.Every(minInterval), 1)
```

Comment:
this is much simpler and easier to understand that the boundedfrequency runner.

The ratelimiter allows one execution per 5 seconds, and the triggers are a netlink event or a maxInterval event